app/src/main/java/org/eyeseetea/malariacare/domain/usecase/push/PushUseCase.java

```diff
@@ -196,6 +162,14 @@ private void banOrgUnit() {
         }
     }
 
+    public IPushController getPushController() {
```

Comment:
If we pass PushController and OrganisationUnitRepository as arguments to strategy constructor this getters is not necessary